### PR TITLE
v1.30.0 - Added alcohol age icon

### PR DESCRIPTION
**Pull request by smilburn1990**
### Added
- Alcohol age icon

### CODE REVIEW

File: src/img/icons/general/alcoholAge.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><defs><style>.cls-1{fill:#d50525;}.cls-2{fill:#fff;}</style></defs><title>18_alcohol_outline</title><rect class="cls-1" width="20" height="20" rx="2"/><path class="cls-2" d="M7.82,14.58H6.46V6.92L4.29,8.25V6.78L6.57,5.42H7.82Z"/><path class="cls-2" d="M15.71,12.14A2.24,2.24,0,0,1,14.82,14a3.57,3.57,0,0,1-2.27.7A3.46,3.46,0,0,1,10.34,14a2.2,2.2,0,0,1-.87-1.82A2.21,2.21,0,0,1,11,10a2.34,2.34,0,0,1-1-.82,2.09,2.09,0,0,1-.37-1.24A2.35,2.35,0,0,1,10.48,6a3.13,3.13,0,0,1,2.07-.7A3.24,3.24,0,0,1,14.66,6a2.34,2.34,0,0,1,.85,2,2.11,2.11,0,0,1-1.37,2A2.11,2.11,0,0,1,15.71,12.14Zm-4.84,0a1.26,1.26,0,0,0,.43,1,1.83,1.83,0,0,0,1.26.39,1.86,1.86,0,0,0,1.31-.42,1.38,1.38,0,0,0,.46-1,1.35,1.35,0,0,0-.45-1,1.87,1.87,0,0,0-1.32-.42,1.76,1.76,0,0,0-1.27.45A1.53,1.53,0,0,0,10.87,12.16ZM11,7.91A1.36,1.36,0,0,0,11.46,9a1.72,1.72,0,0,0,1.14.37A1.61,1.61,0,0,0,13.69,9a1.41,1.41,0,0,0,.43-1.09,1.41,1.41,0,0,0-.39-1,1.56,1.56,0,0,0-1.18-.41,1.48,1.48,0,0,0-1.12.42A1.42,1.42,0,0,0,11,7.91Z"/></svg>
```
Comment (DamianMullins) [inline src/img/icons/general/alcoholAge.svg]: Can you [run this through SVGOMG](https://jakearchibald.github.io/svgomg/) to strip out any unnecessary content, like the title, please?

Comment (smilburn1990) [reply src/img/icons/general/alcoholAge.svg]: @DamianMullins - That better?

File: src/img/icons/general/alcoholAge.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><defs><style>.cls-2{fill:#fff}</style></defs><rect width="20" height="20" rx="2" fill="#d50525"/><path class="cls-2" d="M7.82 14.58H6.46V6.92L4.29 8.25V6.78l2.28-1.36h1.25zM15.71 12.14a2.24 2.24 0 01-.89 1.86 3.57 3.57 0 01-2.27.7 3.46 3.46 0 01-2.21-.7 2.2 2.2 0 01-.87-1.82A2.21 2.21 0 0111 10a2.34 2.34 0 01-1-.82 2.09 2.09 0 01-.37-1.24A2.35 2.35 0 0110.48 6a3.13 3.13 0 012.07-.7 3.24 3.24 0 012.11.7 2.34 2.34 0 01.85 2 2.11 2.11 0 01-1.37 2 2.11 2.11 0 011.57 2.14zm-4.84 0a1.26 1.26 0 00.43 1 1.83 1.83 0 001.26.39 1.86 1.86 0 001.31-.42 1.38 1.38 0 00.46-1 1.35 1.35 0 00-.45-1 1.87 1.87 0 00-1.32-.42 1.76 1.76 0 00-1.27.45 1.53 1.53 0 00-.42 1.02zM11 7.91A1.36 1.36 0 0011.46 9a1.72 1.72 0 001.14.37A1.61 1.61 0 0013.69 9a1.41 1.41 0 00.43-1.09 1.41 1.41 0 00-.39-1 1.56 1.56 0 00-1.18-.41 1.48 1.48 0 00-1.12.42 1.42 1.42 0 00-.43.99z"/></svg>
```
Comment (DamianMullins) [inline src/img/icons/general/alcoholAge.svg]: Sorry, can you remove the width and height attributes as well, please. They cause issues with setting the size via CSS.

Comment (smilburn1990) [reply src/img/icons/general/alcoholAge.svg]: @DamianMullins - Done

**Pull request merged**